src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -0,0 +1,232 @@
+import {LightningElement, api, track} from 'lwc';
+import getDataImportModel
+    from '@salesforce/apex/BGE_DataImportBatchEntry_CTRL.getDataImportModel';
+import getDataImportRows
+    from '@salesforce/apex/BGE_DataImportBatchEntry_CTRL.getDataImportRows';
+import GeFormService from 'c/geFormService';
+import STATUS_FIELD from '@salesforce/schema/DataImport__c.Status__c';
+import FAILURE_INFORMATION_FIELD
+    from '@salesforce/schema/DataImport__c.FailureInformation__c';
+import {deleteRecord} from 'lightning/uiRecordApi';
+import {handleError} from 'c/utilTemplateBuilder';
+import runBatchDryRun from '@salesforce/apex/BGE_DataImportBatchEntry_CTRL.runBatchDryRun';
+
+export default class GeBatchGiftEntryTable extends LightningElement {
+    @api batchId;
+    @track ready = false;
+
+    _batchLoaded = false;
+    @track data = [];
+    @track hasData;
+
+    _columnsLoaded = false;
+    @track columns = [];
+    _columns = [
+        {label: 'Status', fieldName: STATUS_FIELD.fieldApiName, type: 'text'},
+        {label: 'Errors', fieldName: FAILURE_INFORMATION_FIELD.fieldApiName, type: 'text'},
+        {
+            label: 'Donor', fieldName: 'donorLink', type: 'url',
```

Comment:
Do "Donor" and "Donation" need to be custom labels so that we can translate them?

Reply:
Yes, probably - good catch!  I'll make this change now.

Reply:
Updated!